package.json

```diff
@@ -10,11 +10,22 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.5",
+    "@fontsource/roboto": "^5.0.12",
     "@fortawesome/fontawesome-free": "^6.5.2",
```

Comment:
We can remove fontawesome since we already have another package installed such as icons-materias